sunpy/coordinates/tests/test_transformations.py

```diff
@@ -89,14 +89,15 @@ def test_hpc_hpc_null():
 
 def test_hcrs_hgs():
     # Get the current Earth location in HCRS
-    now = Time(parse_time('now'))
-    earth_hcrs = SkyCoord(get_body_barycentric('earth', now), frame='icrs', obstime=now).hcrs
+    adate = Time(parse_time('2015/05/01 01:13:00'))
```

Comment:
We don't need the `Time(` here now right? `parse_time` returns a `Time` now.

Reply:
Good point.